packages/ui/src/services/color-config/color-functions.ts

```diff
@@ -6,14 +6,14 @@ export const colorToRgba = (color: ColorInput, opacity: number) => {
   return new ColorTranslator(color).setA(opacity).RGBA
 }
 
-export const isLightBackground = (color: ColorInput) => {
+export const isLightBackground = (color: ColorInput, threshold: number) => {
```

Comment:
Mb we want to set threshold default value to 120 here in purpose not to pass it everywhere?